CRM/Core/Smarty.php

```diff
@@ -299,12 +299,17 @@ public static function setRequiredTabTemplateKeys(array $tabs): array {
       'count' => NULL,
       'hideCount' => FALSE,
       'template' => NULL,
+      'active' => TRUE,
+      'valid' => TRUE,
       // Afform tabs set the afform module and directive - NULL for non-afform tabs
       'module' => NULL,
       'directive' => NULL,
     ];
 
     foreach ($tabs as $i => $tab) {
+      if (empty($tab['url'])) {
+        $tab['url'] = $tab['link'] ?? '';
```

Comment:
@jensschuppe - sorry about this! I was trying to retain compatibility for whichever of the different existing keys callers were using. But I'm guessing this isn't getting called for the tabsets in question?

Reply:
Right, this has not yet been called, but thankfully with this method (`\CRM_Core_Smarty::setRequiredTabTemplateKeys()`) that's an easy fix.